scripts/run-integration-tests.sh

```diff
@@ -30,7 +30,7 @@ docker ps -a
 
 export ORG_HYPERLEDGER_FABRIC_SDK_DIAGNOSTICFILEDIR=target/diagDump
 export MAVEN_OPTS="-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
-mvn -B clean install -DskipITs=false -Dmaven.test.failure.ignore=false javadoc:javadoc
+mvn -B clean install -DskipITs=false -Dmaven.test.failure.ignore=false javadoc:javadoc -DdisableXmlReport=true
```

Comment:
Note that the [disableXmlReport](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#disableXmlReport) option is deprecated so this may not be the ideal solution.